CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
 
 ### Version Scheme [1.i]
 
-There is a three-part version attached to `CycloneDDS` releases.
-The sources define the version parts as `MAJOR`.`MINOR`.`PATCH`.
-The CMake sources configure the compatibility mode to [`SameMajorVersion`](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-version-file), indicating that breaking changes are accompanied by a change to the `MAJOR` version part.
-There is no explicit policy regarding versioning in the documentation for `CycloneDDS`, however there is some information in the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which discusses _Major_, _Minor_, and _Service_ release criteria.
-It is possible that these release categories align align with changes to the three version parts, but this is not explicitly stated in either `CycloneDDS` documentation or in the Eclipse project handbook.
+`CycloneDDS` versioning follows the standard semantic versioning except that major version 0 is also considered stable. `CycloneDDS` was already a stable code base when it was contributed to [The Eclipse Foundation](https://eclipse.org)
```

Comment:
```suggestion
`CycloneDDS` versioning follows the following rules:
```

The issue here is that this is *not* exactly what [Semantic Versioning](https://semver.org) prescribes. Instead, it is an old, proven model for software versioning that has worked well for many systems requiring stable versions to build on with a good sense of the risk involved in taking a new version based on no more than what numbers have been incremented.

Reply:
updated.